src/mbgl/renderer/render_orchestrator.cpp

```diff
@@ -334,6 +335,8 @@ std::unique_ptr<RenderTree> RenderOrchestrator::createRenderTree(
     }
 
     // Update all sources and initialize renderItems.
+    std::vector<bool> updateList(orderedLayers.size());
```

Comment:
I think you should move the comment to above the for-loop, and maybe add a comment to `updateList`?